project/init.sh

```diff
@@ -5,36 +5,52 @@ SOURCE_URL="https://raw.githubusercontent.com/wttech/aemc/v${VERSION}/project"
 
 AEM_WRAPPER="aemw"
 
-AEM_DIR="aem"
-SCRIPT_DIR="${AEM_DIR}/script"
-HOME_DIR="${AEM_DIR}/home"
-DEFAULT_DIR="${AEM_DIR}/default"
+TARGET_AEM_DIR="instance" # change it if you don't want AEMC to be installed in 'aem' folder
```

Comment:
the idea is good; I will handle that

Reply:
The scripts in this PR are probably a good base, but I didn't realise that I'd need to update Go files too to make it configurable there. TBH it's not a killer feature either way, hence I declined the PR.